.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,100 @@
+name: tests with deployed gateway
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  setup-fixtures:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: ./.github/actions/setup-cli
+        with:
+          access-key: ${{ secrets.SCW_ACCESS_KEY }}
+          secret-key: ${{ secrets.SCW_SECRET_KEY }}
+          project-id: ${{ secrets.SCW_DEFAULT_PROJECT_ID }}
+          organization-id: ${{ secrets.SCW_DEFAULT_ORGANIZATION_ID }}
```

Comment:
It's a shame we have to repeat this, but it's worth it to run the setups in parallel. Can't put this in the action as actions don't have access to secrets.